### PR TITLE
Chef 17 support

### LINKED ISSUE #12: fileutils Chef 17 compatibility

### PR DESCRIPTION
**Pull request by MarkGibbons**
# Description

Chef 17 support

## Issues Resolved

#12 

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Review by ramereth — CHANGES_REQUESTED** [review]
The `chef_version` needs to be set to `>= 15.3`

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by MarkGibbons** [comment]
Updated

File: metadata.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
 version          '1.4.0'
 source_url       'https://github.com/sous-chefs/fileutils'
 issues_url       'https://github.com/sous-chefs/fileutils/issues'
+chef_version     '>=15.3'
```
Comment (ramereth) [inline metadata.rb:10]: You have a duplicate entry for `chef_version` on line 2 that needs to be removed please.

Comment (MarkGibbons) [reply metadata.rb:9]: Chef version is fixed.

Comment (ramereth) [reply metadata.rb:9]: You still need to add a space between `>=` and `15.3` please.

**Comment by MarkGibbons** [comment]
Added a space in the chef_version

**Review by ramereth — APPROVED** [review]

**Comment by damacus** [comment]
We can get rid of the Rakefile now we have the delivery toml

**Review by damacus — CHANGES_REQUESTED** [review]
If we're changing this we should update the rep to use delivery rather than Rake

**Comment by MarkGibbons** [comment]
The Rake file is gone.  Delivery is running rspec which is the part I care about. So I think we are good for now.

**Review by ramereth — APPROVED** [review]

**Review by damacus — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [2.0.0](https://github.com/sous-chefs/fileutils/releases/tag/2.0.0)

**Pull request merged**